bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -643,8 +669,10 @@ private int updateFromSchema3toSchema4(int schemaVersionNumber, CaseDbConnection
 				String attributeLabel = resultSet.getString("type_name");
 				if (attributeTypeId < MIN_USER_DEFINED_TYPE_ID) {
 					updateStatement.executeUpdate(
-							"UPDATE blackboard_attribute_types " + //NON-NLS
-							"SET value_type = " + ATTRIBUTE_TYPE.fromLabel(attributeLabel).getValueType().getType() + " " + //NON-NLS
+							"UPDATE blackboard_attribute_types "
+							+ //NON-NLS
+							"SET value_type = " + ATTRIBUTE_TYPE.fromLabel(attributeLabel).getValueType().getType() + " "
+							+ //NON-NLS
```

Comment:
Please reformat the changes above so that the //NON-NLS comments appear on the same line as the quoted strings, we need that for the automation of localization.